lib/update.php

```diff
@@ -340,6 +361,28 @@ public function vimeography_addon_update_message($plugin_basename, $plugin_data)
   }
 
   /**
+   * Add a reminder to update an expiring key
+   * for the installed Vimeography theme / addon.
+   *
+   */
+  public function vimeography_key_expiring_message() {
+    $today = date("Y-m-d");
+    foreach($this->expiring_addons as $expiring_addon){
+      if($expiring_addon->expiration_date < $today){
+        echo '<div class="notice notice-warning"><p>';
+          printf( __('To continue to receive updates and support for %1$s <a href="http://www.vimeography.com/checkout/?edd_license_key=%2$s&download_id=%3$s">renew your license</a>', 'vimeography'), $expiring_addon->product_name, $expiring_addon->activation_key, $expiring_addon->activation_key);
```

Comment:
I looked into this, and came up empty. If it is discussed online, I can't figure out what to search for to dig it up.